source/tutorial/connecting-to-atlas.txt

```diff
@@ -18,6 +18,13 @@ Atlas uses TLS/SSL to encrypt connections and enforces authentication by
 default. This page explains how to connect to Atlas using
 :program:`mongodrdl` and :program:`mongosqld`.
 
+.. note::
+   
+   With the MongoDB Atlas free tier, |bi| cannot use the ``allowDiskUse``
```

Comment:
I'd also like to make the language in the first sentence a little more exact. e.g. ```When connected to a MongoBD Atlas free tier cluster, the |bi| cannot run aggregation pipelines using the ``allowDiskUse`` option``` or something like that.